optuna/integration/mlflow.py

```diff
@@ -281,9 +277,7 @@ def _set_tags(self, trial: optuna.trial.FrozenTrial, study: optuna.study.Study)
                 tags[key] = "{}...".format(value[: max_val_length - 3])
 
         # This sets the tags for MLflow.
```

Comment:
Could we also remove this comment? Seems unnecessary.

Reply:
Nice point. I agree with you.